docs/components/authentication/sign-up.mdx

```diff
@@ -85,12 +85,48 @@ Below is basic implementation of the `<SignUp />` component. You can use this as
   </Tab>
 
   <Tab>
-    ```js filename="sign-up.js"
-    window.Clerk.mountSignUp(
-      document.getElementById("sign-up")
-    );
-    window.Clerk.openSignUp();
-    ```
+    In the example below, the [`mountSignUp()`](/docs/references/javascript/clerk/sign-up#mount-sign-up) method is used to render the `<SignUp />` component to a `<div>` element with `id="sign-up"`.
+
+    To learn more about the methods available for rendering and controlling the `<SignUp />` component in a JavaScript application, see the [JavaScript reference docs](/docs/references/javascript/clerk/sign-up).
```

Comment:
Worth mentioning they can't use these examples without following the installation steps specific to using Clerk with JavaScript: `/docs/references/javascript/overview#installation`

Reply:
I've added some copy to the new commit that alerts users to follow our quickstart ! (not the js overview, as that is outdated and will be getting reworked)